lib/config/express.js

```diff
@@ -61,6 +65,13 @@ module.exports = function (app) {
     }));
   }
 
+  // Configure GAE proxy and health checks. Force HTTPS.
+  app.enable('trust proxy');
+  // app.use(yesHttps({ maxAge: YES_HTTPS_MAX_AGE, includeSubdomains: true, preload: true }));
```

Comment:
- [ ]  Re-enable yes-http once HTTPS is working with hub.gdgx.io on Cloud Endpoints.